apps/speed.c

```diff
@@ -193,8 +193,12 @@ static int do_multi(int multi, int size_num);
 #endif
 
 static const int lengths_list[] = {
-    16, 64, 256, 1024, 8 * 1024, 16 * 1024
-};
+    2,    4,    6,    8,    10,   12,   14,   
```

Comment:
As this set of size-len is used for most of the other test-cases, are you sure it is useful to enlarge it 4 time or more ?

Reply:
I don't think that suggestion is actually to keep this list. I suppose intention is rather to show the circumstance when you can actually notice the difference. BTW, I don't observe anything close to 14% at any length. And if one cares that much about small TLS fragment performance, I would guess it would make more sense to reduce amount of [indirect] calls to ghash. [I'm putting together something.]

Reply:
See #6312...